src/sctp.rs

```diff
@@ -0,0 +1,73 @@
+mod stream;
+
+pub use stream::{Error as StreamError, Stream};
+
+pub mod association {
+    use crate::sctp::{PayloadType, Stream};
+
+    #[derive(Eq, PartialEq, Clone, Debug)]
+    pub enum Error {}
+
+    #[derive(Debug)]
+    pub struct Association;
+
+    impl Association {
+        pub fn open_stream(&self, _id: u16, _payload_type: PayloadType) -> Result<Stream, Error> {
+            todo!()
+        }
+
+        pub fn accept_stream(&self) -> Result<Stream, Error> {
+            todo!()
+        }
+    }
+}
+
+pub use association::{Association, Error as AssociationError};
+
+#[derive(Eq, PartialEq, Copy, Clone, Debug)]
+pub enum PayloadType {
+    WebRTCBinary,
```

Comment:
cargo clippy
warning: name `WebRTCBinary` contains a capitalized acronym
  --> src/sctp.rs:29:5
   |
29 |     WebRTCBinary,
   |     ^^^^^^^^^^^^ help: consider making the acronym lowercase, except the initial letter: `WebRtcBinary`
   |
   = note: `#[warn(clippy::upper_case_acronyms)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#upper_case_acronyms

warning: name `WebRTCBinaryEmpty` contains a capitalized acronym
  --> src/sctp.rs:30:5
   |
30 |     WebRTCBinaryEmpty,
   |     ^^^^^^^^^^^^^^^^^ help: consider making the acronym lowercase, except the initial letter: `WebRtcBinaryEmpty`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#upper_case_acronyms

warning: name `WebRTCDCEP` contains a capitalized acronym
  --> src/sctp.rs:31:5
   |
31 |     WebRTCDCEP,
   |     ^^^^^^^^^^ help: consider making the acronym lowercase, except the initial letter: `WebRtcdcep`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#upper_case_acronyms

warning: name `WebRTCString` contains a capitalized acronym
  --> src/sctp.rs:32:5
   |
32 |     WebRTCString,
   |     ^^^^^^^^^^^^ help: consider making the acronym lowercase, except the initial letter: `WebRtcString`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#upper_case_acronyms

warning: name `WebRTCStringEmpty` contains a capitalized acronym
  --> src/sctp.rs:33:5
   |
33 |     WebRTCStringEmpty,
   |     ^^^^^^^^^^^^^^^^^ help: consider making the acronym lowercase, except the initial letter: `WebRtcStringEmpty`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#upper_case_acronyms

warning: 5 warnings emitted

Reply:
Fixed.